var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -62,7 +62,7 @@ class Flecsi(CMakePackage):
     depends_on('mpi', when='backend=mpi')
     depends_on('mpi', when='backend=legion')
     depends_on('mpi', when='backend=hpx')
-    depends_on('legion+shared+mpi', when='backend=legion')
+    depends_on('legion+shared_libs', when='backend=legion')
```

Comment:
Given that this seems to be the only dependents of `legion` can we:
```suggestion
    depends_on('legion+shared', when='backend=legion')
```
and then change the legion package to use a `shared` variant (for consistency with all the other packages)?

Reply:
perfect! done.